ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/ModelServer.resource

```diff
@@ -175,7 +175,7 @@ Set Model Server Name
 
 Enable External Serving Route
     [Documentation]    Enables the serving route to be accessed externally
-    SeleniumLibrary.Select Checkbox    xpath://input[@id="alt-form-checkbox-route"]
+    SeleniumLibrary.Select Checkbox    xpath://input[@id="alt-form-checkbox-auth"]
```

Comment:
you sure? this new xpath seems to match the one used for authentication that you find in the variable ${TOKEN_AUTH_CHECKBOX_XP}

at least in RHOAI 2.12, the route checkbox has still `path://input[@id="alt-form-checkbox-route"]` as xpath